tests/playbooks/tests_team_exists.yml

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - name: remove the NetworkManager-team package
+      package:
+        name: "NetworkManager-team"
+        state: absent
+
+    - name: "get the rpm package facts"
+      package_facts:
+        manager: "auto"
+
+    - name: "NetworkManager-team checking before team configuration"
+      debug:
+        msg: "NetworkManager-team is not installed before team configuration"
+      when: "'NetworkManager-team' not in ansible_facts.packages"
+
+    - name: "Team interface configuration"
+      include_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          # Specify the team profile
+          - name: team0
+            state: up
```

Comment:
Does this need state: up? Using only persistent_state:present might allow to test this also on Fedora.

Reply:
I updated to using `persistent_state:present`